models/lorenz_96_tracer_advection/readme.rst

```diff
@@ -0,0 +1,142 @@
+  eastward as they grow, encircling the globe after about 14 days.
+
+  We have chosen a model with :math:`J` variables, denoted by:
+
+  .. math::
+
+     X_1, ... , X_j;
+
+  in most of our experiments we have let :math:`J = 40`. The governing
+  equations are:
+
+  .. math::
+
+     dX_j/dt=(X_{j+1}-X_{j-2})X_{j-1}-X_j+F (1)
+
+  for:
+
+  .. math::
+
+     j=1,...,J.
+
+  To make Eq. (1) meaningful for all values of *j* we define:
+
+  .. math::
+
+     X_{-1}=X_{J-1}, X_0=X_J, \& X_{J+1}=X_1,
+
+  so that the variables form a cyclic chain, and may be looked at as values of
+  some unspecified scalar meteorological quantity, perhaps vorticity or
+  temperature, at *J* equally spaced sites extending around a latitude circle.
+  Nothing will simulate the atmosphere's latitudinal or vertical extent.
+
+In this model we are using the Semi-Lagrangian Scheme to model how tracer particles 
+get distributed upstream across the grids by the Lorenz 96 winds
+
+|Plot of 1D Semi-Lagrangian Method|
+
+The figure above describes the implementation of the Semi-Lagrangian scheme in a 
+one dimensional array. The tracer particle in the figure lands on a predefined grid 
+point at t\ :sup:`n+1`. The trajectory of this tracer particle is then integrated 
+backwards by one time step to time t\ :sup:`n`, often landing between grid points. 
+Then, due to advection without diffusion, the concentration of tracer at time
+ t\ :sup:`n+1` is simply the concentration of tracer at time t\ :sup:`n`, which 
+can be determined by interpolating concentrations of the surrounding grids [3]_.
+
+Once the coupled Lorenz 96 and semi-Lagrangian is run with a source of strength
+ 100 units/s and location at grid point one (with exponential sinks present in
+ all grid points), the time evolution is as depicted below:
+
+|Plot of Lorenz 96 Tracer Advection|
+
+For Lorenz 96 Tracer Advection, DART advances the model, gets the model state and 
+metadata describing this state, finds state variables that are close to a given 
+location, and does spatial interpolation for model state variables.
+
+Namelist
+--------
+
+The ``&model_nml`` namelist is read from the ``input.nml`` file. Namelists
+start with an ampersand ``&`` and terminate with a slash ``/``. Character
+strings that contain a ``/`` must be enclosed in quotes to prevent them from
+prematurely terminating the namelist.
+
+.. code-block:: fortran
+
+  &model_nml
+     model_size        = 120,
+     forcing           = 8.00,
+     delta_t           = 0.05,
+     time_step_days    = 0,
+     time_step_seconds = 3600
+  /
+
+Description of each namelist entry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-------------------+----------+-------------------------------------+
+| Item              | Type     | Description                         |
++===================+==========+=====================================+
+| model_size        | integer  | Number of variables in model. The   |
```

Comment:
i might interpret number of variables as either 3, or 40.  maybe the first sentence should be

```suggestion
| model_size        | integer  | Total number of items in the state vector. The   |
```

but that may not be more clear.   feel free to ignore this suggestion.